agenta-web/src/components/AppSelector/AppSelector.tsx

```diff
@@ -8,7 +8,12 @@ import {useAppTheme} from "../Layout/ThemeContextProvider"
 import {CloseCircleFilled} from "@ant-design/icons"
 import TipsAndFeatures from "./TipsAndFeatures"
 import Welcome from "./Welcome"
-import {getOpenAIKey, isAppNameInputValid, isDemo} from "@/lib/helpers/utils"
+import {
+    getSingleLlmProviderKey,
+    isAppNameInputValid,
+    isDemo,
+    llmAvailableProviders,
```

Comment:
Can you please remove this unused import?

Reply:
Done!